server/testutil/environment/environment.go

```diff
@@ -63,6 +65,9 @@ func (te *TestEnv) bufDialer(context.Context, string) (net.Conn, error) {
 	return te.lis.Dial()
 }
 
+// LocalGRPCServer starts a gRPC server with standard BuildBudy filters that uses an in-memory
+// buffer for communication.
+// Call LocalGRPCConn to get a connection to the returned server.
 func (te *TestEnv) LocalGRPCServer() (*grpc.Server, func()) {
 	te.lis = bufconn.Listen(1024 * 1024)
 	grpcOptions := []grpc.ServerOption{
```

Comment:
can you call your new GRPCServer method here instead of duplicating all that code here?